website/docs/api/large-language-models.mdx

```diff
@@ -712,6 +644,53 @@ it's a function of type `Callable[[Iterable[Any]], Iterable[Any]]`, but specific
 implementations can have other signatures, like
 `Callable[[Iterable[str]], Iterable[str]]`.
 
+### Models via REST API {id="models-rest"}
+
+These models all take the same parameters:
```

Comment:
This might be somewhat misleading. Maybe we should single out `config` and emphasize that its content has to depend on the LLM provider?

Reply:
Good point. What about https://github.com/explosion/spaCy/pull/12945/commits/ccc3fc1318824c9006c0273f09a90431f32d3ac3?

Reply:
Sounds good!